website/pages/resolve/quickstart.mdx

```diff
@@ -30,8 +33,9 @@ docker compose up db
 
 This will initialise a Postgres instance in the container.
 
-<Callout type="info" >
-These instructions assume you have a dump of an OMOP-CDM database called `hutch_db.tar`
+<Callout type="info">
+  These instructions assume you have a dump of an OMOP-CDM database called
```

Comment:
```suggestion
  These instructions assume you have a pre-existing OMOP CDM Postgresql database.
```

Reply:
thanks! I have changed it to "These instructions assume you have a pre-existing OMOP CDM Postgresql database
  and it can be called `hutch_db.tar`, for example, as below.", for readers to understand the further ref. of  hutch_db.tar